subsys/bluetooth/mesh/cfg.c

```diff
@@ -137,7 +137,7 @@ uint8_t bt_mesh_priv_beacon_update_interval_get(void)
 
 int bt_mesh_od_priv_proxy_get(void)
 {
-#if IS_ENABLED(CONFIG_BT_MESH_OD_PRIV_PROXY_SRV)
```

Comment:
I guess line 149 requires similar change

Reply:
Thanks, that was another pattern of values to update